pkg/policy/tree.go

```diff
@@ -89,45 +89,58 @@ func canConsume(root *Node, ctx *SearchContext) api.ConsumableDecision {
 
 // AllowsRLocked checks the ConsumableDecision of the given `SearchContext`.
 // Must be called with the Mutex's tree at least RLocked.
```

Comment:
Can you elaborate on what this comment means? Is this saying that this function should be called with Tree t's mutex locked? If that is the case, should this function be exported?